tee-worker/local-setup/.env.dev

```diff
@@ -30,7 +30,7 @@ DISCORD_AUTH_TOKEN=
 ACHAINABLE_AUTH_KEY=
 ONEBLOCK_NOTION_KEY=
 NODEREAL_API_KEY=
-GENIIDATA_API_KEY=142cf1b0-1ca7-11ee-bb5e-9d74c2e854ac
+GENIIDATA_API_KEY=
```

Comment:
Will it affect any of our tests?
I remember @kziemianek mentioned this is the public key

Reply:
it won't affect any tests. The key is always overwritten.